be/src/exec/tablet_info.cpp

```diff
@@ -134,6 +135,8 @@ OlapTablePartitionParam::OlapTablePartitionParam(std::shared_ptr<OlapTableSchema
 OlapTablePartitionParam::~OlapTablePartitionParam() = default;
 
 Status OlapTablePartitionParam::init(RuntimeState* state) {
+    VLOG(2) << "OlapTablePartitionParam partition init:\n" << apache::thrift::ThriftDebugString(_t_param);
```

Comment:
1. Which expressions are supported in the single table sync materialized view?
2. Does the single materialized view use insert overwrite or not?